web_application/src/CreateProfilePge.js

```diff
@@ -0,0 +1,274 @@
+                                    placeholder="relationshipStatus"
+                                    className="input"
+                                >
+                                    <option value="" disabled selected hidden>Relationship Status</option>
+                                    <option value="single">Single</option>
+                                    <option value="inRelationship">In Relationship</option>
+                                    {/* Add more options */}
+                                    <option value="other">Other</option>
+                                </select>
+                            </div>
+                        </div>
+                    </div>
+                    <text className="boxTitle">ANIMAL OWNERSHIP</text>
+                    <div className="formBoxes">
+                        <input
+                            type="text" 
+                            id="animalOwnership" 
+                            value={animalOwnership} 
+                            onChange={handleAnimalOwnershipChange}
+                            placeholder="Animal type and quantity"
+                            className="input"
+                        />
+                    </div>
+                    <text className="boxTitle" style={{marginTop: "15px"}}>ALLERGIES</text>
+                    <div className="formBoxes">
+                        <input
+                            type="text" 
+                            id="allergies" 
+                            value={allergies} 
+                            onChange={handleAllergiesChange}
+                            placeholder="Please enter your allergies"
+                            className="input"
+                        />
+                    </div>
+                    <text className="boxTitle" style={{marginTop: "15px"}}>HOBBIES</text>
+                    <div className="formBoxes">
+                        <input
+                            type="text" 
+                            id="hobbies" 
+                            value={hobbies} 
+                            onChange={handleHobbiesChange}
+                            placeholder="Please enter your hobbies"
+                            className="input"
+                        />
+                    </div>
+                    <text className="boxTitle" style={{marginTop: "15px"}}>BIO</text>
+                    <div className="bigBox">
+                        <input
+                            type="text" 
+                            id="bio" 
+                            value={bio} 
+                            onChange={handleBioChange}
+                            placeholder="Please enter short BIO"
+                            className="input"
+                        />
+                    </div>
+                </div>
+                <button className="buttons">Create Profile</button>
```

Comment:
should be 'save profile' or something like this as this page will be used also for updating information